hack/lib/util/golang.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+#
+# This library holds golang related utility functions.
+
+# os::golang::verify_go_version ensure the go tool exists and is a viable version.
+function os::golang::verify_go_version() {
+  if ! which go &>/dev/null; then
```

Comment:
*my* scripts should be using a tab... the old `hack/util.sh` crap uses a different thing every ten lines ...